pkg/console/install_panels.go

```diff
@@ -2103,6 +2112,18 @@ func addInstallPanel(c *Console) error {
 				}
 			}
 
+			isDefaultRouteExist, err := checkDefaultRoute()
+			if err != nil {
+				logrus.Error(err)
+				printToPanel(c.Gui, "Failed to check default route.", installPanel)
+				return
+			}
+			if !isDefaultRouteExist {
+				logrus.Error("No default route found. Please check the router setting on the DHCP server.")
+				printToPanel(c.Gui, "No default route found. Please check the router setting on the DHCP server.", installPanel)
```

Comment:
There are three places that use this very same sentence. It would be nice if we made it a const.